app/models/measure.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class Measure < VersionedRecord
+  TASK_MEASURETYPE_ID = 5
```

Comment:
this should probably be better stored elsewhere

or maybe even better store this in a new field in the "measuretypes" table? e.g. `t.boolean send_email_on_update`

Reply:
ooh yeah stored in a column would make way more sense!!

Reply:
we could call it `notifications` like we have on measures